src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Ref.sfxproj

```diff
@@ -14,7 +14,7 @@
   <Target Name="AddFrameworkFilesToPackage" DependsOnTargets="ResolveLibrariesFromLocalBuild" BeforeTargets="GetFilesToPackage">
     <ItemGroup>
       <ReferencePath Include="@(LibrariesRefAssemblies)" />
-      <DocFilesToPackage Include="$(XmlDocDir)%(LibrariesRefAssemblies.FileName).xml" Condition="Exists('$(XmlDocDir)%(LibrariesRefAssemblies.FileName).xml')"/>
+      <DocFilesToPackage Include="$(MicrosoftNetCoreAppRefPackRefDir)%(LibrariesRefAssemblies.FileName).xml" Condition="Exists('$(MicrosoftNetCoreAppRefPackRefDir)%(LibrariesRefAssemblies.FileName).xml')"/>
```

Comment:
It was pointed out in the reported issue that [source build caught this problem](https://github.com/dotnet/runtime/issues/87585#issuecomment-1593009342). What I do not understand from that comment is if source build actually threw an error, or if someone noticed the missing files when manually inspecting.

Either way, is there is a way for us to verify in this repo that the expected xml files are located in their expected location, and throw an error if we don't find them, instead of waiting for source build to catch these problems?

Reply:
> It was pointed out in the reported issue that https://github.com/dotnet/runtime/issues/87585#issuecomment-1593009342. What I do not understand from that comment is if source build actually threw an error, or if someone noticed the missing files when manually inspecting.

@MichaelSimons @NikolaMilosavljevic would know that.

> Either way, is there is a way for us to verify in this repo that the expected xml files are located in their expected location, and throw an error if we don't find them, instead of waiting for source build to catch these problems?

Not that I know. The number of xml files isn't equal to the number of assemblies in the targeting pack. There are files in the targeting pack that are intentionally not documented (i.e. shims). We don't encode which assemblies are shims and which aren't so we can't reliably find the list of assemblies that should have a documentation file.